src/compat/eme/get_webkit_fairplay_initdata.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  bytesToUTF16Str,
+  itole4,
+  le4toi,
+  strToUTF16Array,
+} from "../../utils/byte_parsing";
+
+/**
+ * Create formatted fairplay initdata for WebKit createSession.
+ * Layout is :
+ * [initData][4 byte: idLength][idLength byte: id]
+ * [4 byte:certLength][certLength byte: cert]
+ * @param {Uint8Array} initData
+ * @param {Uint8Array} serverCertificate
+ * @returns {Uint8Array}
+ */
+export default function getWebKitFairPlayInitData(
+  initDataBytes: Uint8Array|ArrayBuffer,
+  serverCertificateBytes: Uint8Array|ArrayBuffer
+): Uint8Array {
+  const initData = initDataBytes instanceof Uint8Array ? initDataBytes :
+                                                         new Uint8Array(initDataBytes);
+  const serverCertificate = serverCertificateBytes instanceof Uint8Array ?
+    serverCertificateBytes :
+    new Uint8Array(serverCertificateBytes);
+  const length = le4toi(initData, 0);
+  if (length + 4 !== initData.length) {
+    throw new Error("Unsupported WebKit initData.");
+  }
+  const initDataUri = bytesToUTF16Str(initData);
+  const skdIdx = initDataUri.indexOf("skd://");
+  const contentIdStr = initDataUri.substring(skdIdx);
```

Comment:
Here we still include it :p 
Does this work ?

Also, the fact that the original code was something like `.split("skd://")[1]` makes me thinks that it is always at first position.

_Now that I'm thinking of it, the old monorepo code would be good if `initDataUri` could represent multiple URI concatenated but I heavily doubt it._

If that skd thing is always found first and if there's only one URI, maybe the more understandable way would be to do something like:
```js
contentIdStr = startsWith(initDataUri, "skd://") ? initDataUri.substring(6) :
                                                   initDataUri;
```